usr/src/lib/libict_pymod/ict.py

```diff
@@ -1018,8 +1020,13 @@ def generate_sc_profile(self):
             return 0
 
         info_msg('Detected ' + self.keyboard_layout + ' keyboard layout')
-        status = _cmd_status('/usr/bin/sed s/US-English/' + \
+
+        if os.path.isfile(self.basedir + '/usr/sbin/lightdm'):
+            display_manager='lightdm' 
+            
+        status = _cmd_status('/usr/bin/sed -e s/US-English/' + \
```

Comment:
Why is this needed? Comment would help. I suppose this is related to gdm pieces being in illumos-gate?